frontend/src/components/bom/BOMDetailView.jsx

```
@@ -0,0 +1,2313 @@
+      work_center_id: "",
+      operation_name: "",
+      run_time_minutes: "0",
+      setup_time_minutes: "0",
+    });
+    setShowAddOperation(false);
+  };
+
+  const handleRemovePendingOperation = (index) => {
+    const updated = pendingOperations.filter((_, i) => i !== index);
+    // Resequence
+    updated.forEach((op, i) => (op.sequence = i + 1));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)